web/src/components/storage/DriveEditor.tsx

```diff
@@ -380,14 +380,15 @@ const SearchSelector = ({ drive, selected, onChange }) => {
 
 const RemoveDriveOption = ({ drive }) => {
   const driveModel = useDrive(drive.name);
+  const { hasAdditionalDrives } = useModel();
 
   if (!driveModel) return;
 
   const { isExplicitBoot, delete: deleteDrive } = driveModel;
 
+  if (!hasAdditionalDrives) return;
```

Comment:
When `hasAdditionalDrives` no button is displayed. That's correct IMHO.

But the cases of `hasPv` and `isExplicitBoot` should actually display a disabled button with the corresponding explanation. I forgot to add a comment to the code explaining all that.

Reply:
Comments added